tools/plugins/com.liferay.ide.gradle.core/src/com/liferay/ide/gradle/core/LiferayGradleWorkspaceProjectProvider.java

```diff
@@ -158,7 +158,7 @@ private static IWorkspaceProject getWorkspaceProjectFromLiferayHome(final IPath
 			LiferayWorkspaceUtil.getWorkspaceProject()
 		).filter(
 			workspaceProject -> {
-				IPath workspaceLocation = workspaceProject.getRawLocation();
+				IPath workspaceLocation = workspaceProject.getLocation();
```

Comment:
location is the workspace relative location and it will never be a prefix of bundle location since that is the physical location.